webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalSelectors.js

```diff
@@ -1,7 +1,11 @@
+import Immutable from 'seamless-immutable';
+
 export const selectForemanModalsState = state => state.foremanModals;
 export const selectModalStateById = (state, id) =>
-  state.foremanModals && state.foremanModals[id];
-export const selectIsModalOpen = (state, id) => {
-  const openState = selectModalStateById(state, id);
-  return openState && openState.open;
-};
+  state.foremanModals && (state.foremanModals[id] || Immutable({}));
```

Comment:
why creating an Immutable empty object?  selectors don't affect redux's store, just read from it.

Reply:
So I do not get `undefined` state when checking if modal is open:

![undefined-open](https://user-images.githubusercontent.com/2664090/75433658-94fbb100-5950-11ea-8e15-ba5e226ffe15.png)

Reply:
I mean using `Immutable` here is overkill,  an empty object will do the job :)
```js
state.foremanModals && (state.foremanModals[id] || {})
```

Reply:
Updated.